CHANGELOG.md

```diff
@@ -13,6 +13,10 @@ This changelog documents all notable user-facing changes of VAST.
 
 ## Unreleased
 
+- 🐞 The JSON parser now accepts data with numerical or boolean values in
```

Comment:
I would make this a change instead of bug. Our parser behaved correctly beforehand, but now it operates in a "enhanced" mode and accepts inputs more liberally.

Reply:
I disagree. From a users perspective, this is a bug. We do all kinds of conversions here and this one was missing.

Reply:
Okay, I have not a particularly strong opinion here.